speech_to_text.py

```diff
@@ -118,7 +122,13 @@ def run_whisper(job: dict) -> dict:
         whisper_options.pop("model", None)
         whisper_options.pop("writer", None)
 
-        result = whisper.transcribe(audio=media_file, model=model, **whisper_options)
+        try:
+            result = whisper.transcribe(
+                audio=media_file, model=model, **whisper_options
+            )
+        except Exception as e:
+            raise SpeechToTextException(str(e))
+
```

Comment:
I introduced a new exception class, so I wanted to make sure we were catching all Whisper exceptions and wrapping them in it.